api/src/task/forms/list_forms_task.py

```diff
@@ -75,14 +75,16 @@ def process_form(self, form: Form) -> None:
 
             # Add a message to the out-of-date forms list so we can make it easier
             # to update those forms by generating the command with some messages for organization
-            self.out_of_date_forms.extend(
-                [
-                    f"# {form_txt}",
-                    "# WARNING: If this form has a form_instruction record you'll need to create that"
-                    "# and add it to the command below manually.\n",
-                    update_cmd,
-                ]
-            )
+            out_of_date_entries = [f"# {form_txt}", update_cmd]
+
+            # If the form has a form_instruction_id, also include the command to update the instruction
+            if form.form_instruction_id is not None:
+                form_instruction_cmd = get_update_form_instruction_cmd(
+                    self.environment, form_id, str(form.form_instruction_id)
+                )
+                out_of_date_entries.append(form_instruction_cmd)
+
+            self.out_of_date_forms.extend(out_of_date_entries)
```

Comment:
The other chunk that says when a form is out-of-date, but exists has a divider line that I was missing on this case, so it makes things run together a bit without it. I think if we add this, they'd match.

```suggestion
            out_of_date_entries = [f"# {form_txt}", update_cmd]

            # If the form has a form_instruction_id, also include the command to update the instruction
            if form.form_instruction_id is not None:
                form_instruction_cmd = get_update_form_instruction_cmd(
                    self.environment, form_id, str(form.form_instruction_id)
                )
                out_of_date_entries.append(form_instruction_cmd)
            
            out_of_date_entries.append("\n---")

            self.out_of_date_forms.extend(out_of_date_entries)
```

What it looks like at the moment (training is missing a lot of forms, can point to that to see this).

<img width="1435" height="193" alt="Image" src="https://github.com/user-attachments/assets/aa3c3637-818b-4e14-b3eb-7699be3086bd" />